data/json/items/books.json

```diff
@@ -3096,7 +3097,8 @@
         "fun" : -1,
         "skill" : "none",
         "price" : 1500,
-        "required_level" : 0
```

Comment:
We should probably toughen the requirements for reading the priest's diary as Latin isn't exactly common knowledge.  ('Twas a quest item, IIRC.)